models/migrations/v1_21/v270.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_21 //nolint
+
+import (
+	"xorm.io/xorm"
+)
+
+func FixPackagePropertyTypo(x *xorm.Engine) error {
+	sess := x.NewSession()
+	defer sess.Close()
+
+	if err := sess.Begin(); err != nil {
+		return err
+	}
```

Comment:
`db.TxContext()`?

Reply:
We never used db.TxContext in migrations. If we want to support that, we need to refactor the migration interface first which should not be in this PR.